include/app/config.hpp

```diff
@@ -162,6 +166,7 @@ class Config : public QObject {
     QString cam_local_device;
     QString cam_name;
     bool cam_is_network;
+    QVideoFrame::PixelFormat cam_local_format_override;
```

Comment:
what does this look like in the config file? I'm assuming an integer since its an enum?

My only concern with an enum is that they values may change between Qt versions, but also cant really think of a better way to store it lol

Reply:
I agree in theory but in practice it's highly unlikely because they would probably break a bunch of apps.

Reply:
See below comment for example